<?xml version="1.0"?>
<launch>
  <node name="gpsd_conv" pkg="gps_common" type="utm_odometry_node">
    <!-- remap from="odom" to="vo1"/ --> <!-- if two gpss used, otherwise -->
    <remap from="odom" to="vo" />
    <remap from="fix" to="fix" />  
    <param name="rot_covariance" value="99999" />
    <param name="frame_id" value="base_footprint" />
  </node>
</launch>

